tutorials/configuring-loadbalancer-wordpress/index.mdx

```diff
@@ -18,9 +18,9 @@ The capacity of a single server is limited. Once a website gains more and more a
 
 This is the point where a Load Balancer enters the game. It allows spreading the "load" that all those visitors and their requests create to be "balanced" over a series of different Instances.
 
-In case of increasing load on a setup, capacity can easily be increased by adding more Instances to the load balancers backend. This allows to scale your infrastructure without any downtime or delays whilst waiting for DNS zones to be updated.
+In case of increasing load on a setup, capacity can easily be increased by adding more Instances to the load balancer backend. This allows you to scale your infrastructure without any downtime or delays whilst waiting for DNS zones to be updated.
```

Comment:
```suggestion
In case of increasing load on a setup, capacity can easily be increased by adding more Instances to the Load Balancer's backend. This allows you to scale your infrastructure without any downtime or delays whilst waiting for DNS zones to be updated.
```